internal/db/migrations/2_alter_table_rename_account.down.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE composes RENAME COLUMN account_number TO account_id;
```

Comment:
Should we also test migrating down ?

Reply:
They're there mostly for completeness and potential future debugging of db issues. We never migrate a live database down, only up. (So say a mistake in a migration.sql file got made, it would have to be fixed by an additional up migration if at all possible).

Reply:
I see, okay, it makes sense.